CONTRIBUTING.rst

```diff
@@ -18,12 +18,71 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
 developing at least a prototype reference implementation of your idea.
 
 
+Contributing changes to existing PEPs
+-------------------------------------
+
+In general, PEPs that are not marked as Draft or Active are usually considered
+to be historical documents rather than documentation or living specifications.
+To propose a substantially revised version of the original ideas,
+you should generally consider opening a new PEP instead.
```

Comment:
Though not every change to something specified in a PEP will itself need a PEP to change.

Reply:
Yeah—if this sounds a bit out of place here, that's because it was originally part of the first bullet (Deferred/Withdrawn/Rejected), was more about do-overs for old PEPs. I realized it could apply, in somewhat modified form, to Final and Accepted PEPs too, so I moved it here.

In any case, to address this I completely re-wrote this paragraph to strengthn the conditions for a new PEP, add more alternatives, integrate the sentences better and add "Provisional" to "Active" and "Draft" above.